### ISSUE TITLE
after pressing the view activity log page button, it will show new page everytime

### ISSUE THREAD

**Issue by himanshujaidka** [mozilla/extension-activity-monitor#60]
it needs to give output that the page is already opened 
![Screenshot (85)](https://user-images.githubusercontent.com/58654018/94537036-8ec01100-0260-11eb-85fa-fd0aaa5beac3.png)

**Comment by himanshujaidka** [comment]
even this issue is with the current version also

**Comment by atiqueahmedziad** [comment]
Currently, this is the expected behavior. I can think of one use case, which may be supported by opening a new instance of activity log page (with realtime logs) every time it is chosen from the popup. 
The user case:
- The user may want to open another instance of activity logs page (with realtime logs) to apply a different set of filters comparing to the first instance of the activity log page. 

I am not sure whether this use case is valid or not from actual user's (API developers, Reviewers) perspective. 

@wagnerand @rpl @Rob--W any thought regarding this?

**Comment by wagnerand** [comment]
@atiqueahmedziad Do all pages share the same filter settings?

**Comment by atiqueahmedziad** [comment]
> @atiqueahmedziad Do all pages share the same filter settings?

No, the filter settings are not shared. Th filter settings are separated with every instance of activity log page. Any new activity log page will have zero filter applied (which means all the existing logs will be showed initially).

**Comment by wagnerand** [comment]
> > @atiqueahmedziad Do all pages share the same filter settings?
> 
> No, the filter settings are not shared. The filter settings are separated with every instance of the activity log page. Any new activity log page will have zero filter applied (which means all the existing logs will be showed initially).

Thanks! I think we could keep the current behavior then, to support the use case you mentioned earlier.
